src/deps-index.ts

```diff
@@ -13,12 +13,14 @@
  */
 import {AssertionError} from 'assert';
 import {Analyzer} from 'polymer-analyzer';
+import * as urlLib from 'url';
 
+import * as urlUtils from './url-utils';
 import {UrlString} from './url-utils';
 
 export interface DepsIndex {
   // An index of entrypoint -> html dependencies
-  entrypointToDeps: Map<string, Set<string>>;
+  entrypointToDeps: Map<UrlString, Set<UrlString>>;
```

Comment:
I'm not sure how much `UrlString` is buying us, since it's a structural interface. We should reconsider this, but not in this PR.